lib/dash/mpd_utils.js

```diff
@@ -225,8 +225,7 @@ shaka.dash.MpdUtils = class {
         timeline[timeline.length - 1].end = startTime / timescale;
       }
 
-      for (const _ of shaka.util.Iterables.range(repeat + 1)) {
-        shaka.util.Functional.ignored(_);
+      for (let j = 0; j <= repeat; ++j) {
```

Comment:
For historical context, we used to have frequent issues with mistakes in code like this.  It is easy to mix up `i` and `j` for example, and easy to overlook in code review.  So `Iterables` was a way to avoid the need for those.

Clearly the performance penalty is too great, though.  We should figure out what other things transpile into poorly-performing ES5 and eliminate them.  We could even write a test that would check the binary for poorly-performing polyfills, to catch regressions.

Reply:
From Monday I will continue working on other improvements, but I will create other PRs.